src/test/kotlin/g1601_1700/s1608_special_array_with_x_elements_greater_than_or_equal_x/SolutionTest.kt

```diff
@@ -0,0 +1,22 @@
+package g1601_1700.s1608_special_array_with_x_elements_greater_than_or_equal_x
+
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert
+import org.junit.jupiter.api.Test
+
+internal class SolutionTest {
+    @Test
+    fun specialArray() {
+        MatcherAssert.assertThat(Solution().specialArray(intArrayOf(3, 5)), CoreMatchers.equalTo(2))
```

Comment:
Pleased replace `MatcherAssert.assertThat(` with `assertThat(`. Ctrl - Shift - r in Idea